models/DataObject/ClassDefinition/DynamicOptionsProvider/MultiSelectOptionsProviderInterface.php

```diff
@@ -18,6 +18,11 @@
 
 use Pimcore\Model\DataObject\ClassDefinition\Data;
 
+trigger_deprecation('pimcore/pimcore', '11.1', '%s is deprecated. Use %s instead.', MultiSelectOptionsProviderInterface::class, SelectOptionsProviderInterface::class);
```

Comment:
This needs to be `11.2` now, since `11.1` has already been released.

```suggestion
trigger_deprecation('pimcore/pimcore', '11.2', '%s is deprecated. Use %s instead.', MultiSelectOptionsProviderInterface::class, SelectOptionsProviderInterface::class);
```